commands/inventory/add_service_mysql.go

```diff
@@ -93,8 +119,9 @@ var (
 func init() {
 	AddServiceMySQLC.Arg("name", "Service name").StringVar(&AddServiceMySQL.ServiceName)
 	AddServiceMySQLC.Arg("node-id", "Node ID").StringVar(&AddServiceMySQL.NodeID)
-	AddServiceMySQLC.Arg("address", "Address").StringVar(&AddServiceMySQL.Address)
-	AddServiceMySQLC.Arg("port", "Port").Int64Var(&AddServiceMySQL.Port)
+	AddServiceMySQLC.Arg("address", "Address").Default("").StringVar(&AddServiceMySQL.Address)
+	AddServiceMySQLC.Arg("port", "Port").Default("").Int64Var(&AddServiceMySQL.Port)
```

Comment:
Why we need an empty string as the default value for a port?

Reply:
To not make it required field and differentiate when user set the value "127.0.0.1:3306" manually